circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/CopierFactoryManagerTest.java

```diff
@@ -15,19 +15,22 @@
  */
 package com.hotels.bdp.circustrain.core;
 
-import static org.junit.Assert.assertTrue;
-import static org.mockito.Mockito.when;
-
-import java.util.Arrays;
-
+import com.google.common.collect.ImmutableMap;
```

Comment:
Again, minor but order of imports should be
```
java
javax
org
*
com
com.hotels
```
(we need to put this info somewhere on GH and add it to our contributing guidelines)